re.sonny.Workbench.json

```diff
@@ -0,0 +1,102 @@
+{
+  "app-id": "re.sonny.Workbench",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "41",
+  "sdk": "org.gnome.Sdk",
+  "command": "workbench",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "libadwaita",
+      "buildsystem": "meson",
+      "config-opts": ["-Dvapi=false", "-Dexamples=false", "-Dtests=false"],
+      "cleanup": ["/include", "/lib/pkgconfig"],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libadwaita/1.0/libadwaita-1.0.0.alpha.4.tar.xz",
```

Comment:
1.0.2 is out, is there something wrong with it?

Reply:
It's not a blocker, please take a look at this after the merge.

Reply:
requires a newer gtk version